tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1432,6 +1431,47 @@ describe("insertTestResult", () => {
             testResultsService = new TestResultsService(new MockTestResultsDAO());
 
             return testResultsService.insertTestResult(testResultWithAdrTestTypeWithoutExpiryDate)
+                .then((data: any) => {
+                    expect(data).not.be.eql(undefined);
+
+                })
+                .catch((error: { statusCode: any; body: any; }) => {
+                    expect.fail();
+                });
+        });
+    });
+
+    context("when inserting a submitted testResult that has an ADR testType without a certificateNumber", () => {
+        it("should throw 400 and descriptive error message", () => {
+            const testResultWithAdrTestTypeWithoutCertificateNumber = testResultsPostMock[6];
```

Comment:
Definitely don't mutate un-cloned instances of global objects. Will potentially be re-used in other tests. Lodash's cloneDeep is your friend here.